common/lib/capa/capa/templates/status_span.html

```diff
@@ -0,0 +1,5 @@
+<%page expression_filter="h" args="status, sr_text, classname, status_id"/>
+<span class="status ${classname}" id="status_${status_id}" data-tooltip="${status.display_tooltip}">
+  <span class="sr">${sr_text}</span>
```

Comment:
That was going to be in my next iteration now that we are not using different sr text for some problems. I will add that change to the next push.